app/views/NewServerView.js

```diff
@@ -182,17 +181,30 @@ export default class NewServerView extends LoggedView {
 					<SafeAreaView style={sharedStyles.container} testID='new-server-view' forceInset={{ bottom: 'never' }}>
 						<Image style={styles.image} source={{ uri: 'new_server' }} />
 						<Text style={styles.title}>{I18n.t('Sign_in_your_server')}</Text>
-						<TextInput
-							inputRef={e => this.input = e}
-							containerStyle={styles.inputContainer}
-							placeholder={defaultServer}
-							value={text}
-							returnKeyType='send'
-							onChangeText={this.onChangeText}
-							testID='new-server-view-input'
-							onSubmitEditing={this.submit}
-							clearButtonMode='while-editing'
-						/>
+						<View
+							style={styles.inputContainer}
+						>
+							<Picker
+								mode='dropdown'
+								selectedValue={protocolType}
+								onValueChange={this.handlePickerValueChange}
+								style={styles.picker}
+							>
+								<Picker.Item label='https://' value='https://' />
```

Comment:
for repeating strings it's better to define constants `protocol.HTTP` and `protocol.HTTPS`